eng/pipelines/templates/BuildAndTest.yml

```diff
@@ -23,13 +23,45 @@ parameters:
 steps:
   - script: ${{ parameters.buildScript }}
             -restore -build
-            -pack
```

Comment:
Won't this mean that public builds won't run pack any longer?

Reply:
They will run them on the `helix tests` job.
```diff
+  - ${{ if and(eq(parameters.runAsPublic, 'true'), eq(parameters.runHelixTests, 'true')) }}:
+    # Build packages for testing
+    - script: ${{ parameters.buildScript }}
+              -restore
+              -pack
+              -configuration ${{ parameters.buildConfig }}
+              /bl:${{ parameters.repoLogPath }}/pack.binlog
+              $(_OfficialBuildIdArgs)
+      displayName: Pack for testing
```

Reply:
similar to the other question, but do we really need to do it in two steps? I think this complicates the build and also makes it such that a lot of 1ES steps have to run multiple times. Could we not condense everything into two build calls? (one for the product and one for the workloads)